app/Controllers/importExportController.php

```diff
@@ -503,9 +509,9 @@ private function importJson($article_file, $starred = false) {
 	 * This method import a JSON-based feed (Google Reader format).
 	 *
 	 * @param array<string,string> $origin represents a feed.
-	 * @return FreshRSS_Feed|null if feed is in database at the end of the process, else null.
+	 * return if feed is in database at the end of the process, else null.
```

Comment:
I think the `@return` is still needed to assign the proper explanation to the return definition

Reply:
the return is clear, i use "?" to inform the result is nullable.

Reply:
What I mean is that there is an error in the PHPDocs syntax, preventing attaching **the description** of the result correctly.

Before your changes (correct):
![image](https://user-images.githubusercontent.com/1008324/223983943-4734b4d3-f248-4827-8e57-7e01f6091b26.png)

After your changes (wrong):
![image](https://user-images.githubusercontent.com/1008324/223984004-af80f859-733f-4884-be25-2384efc4a015.png)

Reply:
done